docs/machine-learning/how-to-guides/machine-learning-model-predictions-ml-net.md

```diff
@@ -131,3 +138,15 @@ The predicted values in the score column should look like the following:
 | 1 | 144638.2 |
 | 2 | 150079.4 |
 | 3 | 107789.8 |
+
+## Multiple predictions (PredictionEnginePool)
+
+To make multiple predictions using [PredictionEnginePool](xref:Microsoft.Extensions.ML.PredictionEnginePool%602), you can take an `IEnumerable` containing multiple instances of your model input. For example and `IEnumerable<HousingInput>` and apply the [`Predict`](xref:Microsoft.Extensions.ML.PredictionEnginePoolExtensions.Predict%2A) method to each element using LINQ's [`Select`](xref:System.Linq.Enumerable.Select%2A) method.
```

Comment:
"For example **_and_** `IEnumerable<HousingInput>` and apply..."

Should this "and" be "an"?

Reply:
Resolved